src/main/java/frc/robot/Constants.java

```diff
@@ -1,76 +1,71 @@
 package frc.robot;
 
-import frc.io.Dashboard;
-import frc.util.pid.PIDConstants;
 import frc.util.pid.SparkMaxConstants;
 
 public class Constants {
-    
     public static final boolean USING_DASHBOARD = true;
     public static final double CONTROLLER_DEADZONE = 0.1;
-    public static final double WhHEEL_DIAMETER = 6;
+    public static final double WHEEL_DIAMETER = 6;
     public static final double MAX_OUTPUT = 1;
     public static final double PATH_TURN_P = 6;
-    
-    public static final int wheelID = 22;
```

Comment:
Please follow for all static final constants the format of all uppercase with _ as spaces. See also: https://geosoft.no/javastyle.html#Best_Practices (Section 2.4)

Reply:
Ok ok...